panther_analysis_tool/constants.py

```diff
@@ -91,3 +91,6 @@ class ReplayStatus:
 
 
 ENABLE_CORRELATION_RULES_FLAG = "EnableCorrelationRules"
+
+CACHE_DIR = ".cache"
```

Comment:
do we want users to commit this? i assume not. let's also add `.cache/` to the .gitignore file in the repo, such that if someone clones it, itll be ignored right away. for existing repos though, we may wish to add a warning if we detect they dont have the `.cache/` directory ignored

Reply:
Good idea. Added to this repo. It gets added to the customer repo during `pat init` though.